packages/proof-service/src/index.ts

```diff
@@ -2,16 +2,8 @@ import { DatabaseEngine } from '@zkdb/storage';
 import TaskService from './service/task-service.js';
 import logger from './helper/logger.js';
 import { config } from './helper/config.js';
-import { Mina } from 'o1js';
 
 (async () => {
-  const network = Mina.Network({
```

Comment:
Does it work if you remove the setActiveInstance thing

Reply:
Moved to create-proof method